src/com/biermacht/brews/frontend/MainActivity.java

```diff
@@ -235,7 +235,9 @@ public void onDrawerOpened(View drawerView) {
     fragmentList.add(new EditMashProfilesFragment());
     fragmentList.add(new ViewStylesFragment());
     fragmentList.add(new StrikeWaterCalculatorFragment());
-    fragmentList.add(new HydrometerTempCalculatorFragment());
+    HydrometerTempCalculatorFragment hydrometerTempCalculatorFragment = new HydrometerTempCalculatorFragment();
```

Comment:
I _think_ that instead of passing in the preferences here, you could do something like this inside of the fragment itself whenever you need to access the preferences:

```
this.getActivity().getSharedPreferences(Constants.PREFERENCES, Context.MODE_PRIVATE)
```

My concern being that, if the fragment is destroyed / re-created by Android at some point, the way it is now it might lose the passed in preferences object and `hydrometerTempCalculatorFragment.preferences` could end up a null pointer.